### DISCUSSION TITLE
WebGPU support planned?

### DISCUSSION THREAD

**Discussion by mmassing** [DiligentGraphics/DiligentEngine#235] [General]
I'd like to use DiligentCore for an upcoming project. Since we may want to target WebAssembly and WebGPU in the future, I was wondering if WebGPU support is a consideration?

**Comment by TheMostDiligent** [comment]
Yes, WebGPU is definitely a consideration, but is not in immediate plans at the moment.
Diligent currently supports WebGL though.

**Reply by mmassing** [reply]
Thanks for the reply, that's great to hear! I think this also answers my implicit question, whether it seems feasible.
